### PR TITLE
Modify UI for TABS

### PR DESCRIPTION
**Pull request by Aloynz**
Changes:
- More background colours adopted
- Icons for phone, address and email of persons added
- Add numbering and fix styling for groups list

image

### CODE REVIEW

File: src/main/java/seedu/address/ui/PersonCard.java [hunk 1]
```diff
@@ -54,9 +61,22 @@ public PersonCard(Person person, int displayedIndex) {
         this.person = person;
         id.setText(displayedIndex + ". ");
         name.setText(person.getName().fullName);
-        phone.setText(person.getPhone().value);
-        address.setText(person.getAddress().value);
-        email.setText(person.getEmail().value);
+        if (person.getPhone() != null) {
+            phone.setText(person.getPhone().value);
```
Comment (zylee348) [inline src/main/java/seedu/address/ui/PersonCard.java:65]: good to check for null case

File: src/main/resources/view/DarkTheme.css [hunk 2]
```diff
@@ -344,10 +344,10 @@
 
 #tags .label {
     -fx-text-fill: white;
-    -fx-background-color: #3e7b91;
+    -fx-background-color: rgb(0, 0, 199);
```
Comment (zylee348) [inline src/main/resources/view/DarkTheme.css:347]: good use of rgb rather than the standard hex

**Review by zylee348 — COMMENTED** [review]
LGTM overall

File: src/main/java/seedu/address/ui/PersonCard.java (hunk 1 above)
Comment (LianGuoYang) [reply src/main/java/seedu/address/ui/PersonCard.java:65]: We already have a null case check for all of the variables in Person class, is this necessary?

**Review by LianGuoYang — COMMENTED** [review]
LGTM

**Comment by Tan-Jin-Waye** [comment]
LGTM

**Pull request merged**